src/main/java/tk/shanebee/hg/game/GamePlayerData.java

```diff
@@ -305,6 +307,20 @@ public void join(Player player, boolean command) {
                 freeze(player);
                 kills.put(player, 0);
 
+                ItemStack bed = new ItemStack(Material.RED_BED, 1);
+                ItemMeta commeta = bed.getItemMeta();
+                commeta.setDisplayName("Leave");
+                bed.setItemMeta(commeta);
+                player.getInventory().setItem(8, bed);
+
+                if (player.isOp()){
+                    ItemStack start = new ItemStack(Material.NETHER_STAR, 1);
```

Comment:
This item should probably be configurable in the config.
I just have a small feeling once this is added, people will ask for a config option